widget/embedded/src/hooks/useFetchConfig.ts

```diff
@@ -0,0 +1,46 @@
+import { useUiStore } from '../store/ui';
+import { getConfig } from '../utils/configs';
+
+export type Watermark = 'NONE' | 'FULL';
+
+type ConfigResponse = {
+  config: {
+    watermark: Watermark;
+  };
+};
+
+interface useFetchConfig {
```

Comment:
Please use capital letter for first letter of interface names.

Reply:
fixed